android9-generic.yaml

```diff
@@ -28,7 +28,7 @@ actions:
   - action: run
     chroot: true
     description: Create directories for Android mounts
-    command: mkdir -p /android /data /system /system_root /vendor /cache /mnt/vendor/persist /userdata
```

Comment:
I'd prefer to leave them for now unless they break halium-install. This way we do not require additional post-processing for the built rootfs and keep required changes in one place.

Reply:
They do break the symlinks created by halium-install and (in the future) rootstock-ng.